crd/sparkcluster.crd.yaml

```diff
@@ -66,7 +66,14 @@ spec:
                           instances:
                             type: integer
                 image:
```

Comment:
I'm pretty late for this: Do we want to use the term "image" here?
Paging @soenkeliebau 

I'd have thought just "version" is good enough?

Reply:
I would prefer version over image, yes. The image String is something we build internally to use in the pod.